### DISCUSSION TITLE
Switched to PipeWire

### DISCUSSION THREAD

**Discussion by msdobrescu** [mocaccinoOS/mocaccino#71] [General] [answered]
Hello,

For me PulseAudio was broken for years, I had glitches and kept distorting more and more often.
As PipeWire support has been added recently, thanks to @joostruis's work, I've switched to it.
If you are interested in this, just follow Replacing PulseAudio section from [Gentoo's wiki](https://wiki.gentoo.org/wiki/PipeWire).
No need to do anything else than stopping and disabling PulseAudio services and enabling/starting PipeWire's, along with the apps that were using sound/mics at that time.
Also [Arch wiki](https://wiki.archlinux.org/title/PipeWire#Sound_quality_(resampling_quality)) may help with tuning.
The main place to read how to configure it is, of course, [the PipeWire wiki here](https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Config-PipeWire).
I've set it up and tried it for a week now, it did not glitch at all so far! I've successfully used it in conferencing in Teams on Chrome.
Have in mind one thing, as I've read, PipeWire it's still in development, so may change some settings in time, so that may break with some upgrade and may need to fix the setup then.

Hope it helps!

**Comment by joostruis** [comment, marked as answer]
For the record. Since MocaccinoOS beta 6 we switched to Pipewire.
https://github.com/mocaccinoOS/mocaccino/releases/tag/v0.1.0-beta06
